src/persist-client/src/cli/admin.rs

```diff
@@ -664,3 +665,138 @@ async fn force_gc(
 
     Ok(Box::new(machine))
 }
+
+/// WIP
+pub const CATALOG_FORCE_COMPACTION_FUEL: Config<usize> = Config::new(
+    "persist_catalog_force_compaction_fuel",
+    // WIP
+    1024,
```

Comment:
Oh, note to self to flip these defaults back (and then add CI overrides) before merging. I just have them like this right now to make it easy to test on Joe's staging env